integration-test/data/FoD/fod-integation-test-data.json

```diff
@@ -10,8 +10,8 @@
 	"expectOutput": "default"
   },  
   {
-    "commands": "fod application create-web-app",
-    "arguments": "--owner=1851 --business-criticality=High --sdlc-status='Production' INT_T_App2:INT_T_App2_Release2",
+    "commands": "fod application create",
+    "arguments": "--app-type=Web --owner=1851 --business-criticality=High --sdlc-status='Production' INT_T_App2 INT_T_App2_Release2",

```

Comment:
Can you use `FCLI_FOD_USER` instead of hardcoded value for `--owner` here - if you use a user name it will try and look it up.